packages/commonwealth/server/routing/external.ts

```diff
@@ -151,7 +151,6 @@ export function addExternalRoutes(
     deleteEntities.bind(this, 'chain_id', models, models.Topic)
   );
 
-  router.get('/roles', getRolesValidation, getRoles.bind(this, models));
```

Comment:
Feel free to delete this route entirely. We can also remove it from swagger_external.json, so it doesn't display on our route docs.